### PR TITLE
Update navigation for Level 2 access

### PR DESCRIPTION
**Pull request by corypride**
## Description of the change
Admins should see the following navigation menu:

Student Activity → This is the old “Dashboard”

Students → Links to the current Students page

Admins → Links to the current Admins page

Facilities → Links to the current Facilities page

Learning Platforms → This is the Provider Platform page.
Residents should see the following navigation menu:

My Courses → Links to the current My Courses page

My Learning → This is the old dashboard.
- **Related issues**:
Closes: #509

## Screenshot(s)

https://github.com/user-attachments/assets/9f9acbe7-42eb-45c6-8d00-b6745ce5f293

### CODE REVIEW

**Comment by calisio** [comment]
This is going to have a lot of conflicts with my PR #532. I'm not sure what the best approach is for this @PThorpe92

**Comment by PThorpe92** [comment]
@calisio we met today and discussed this. He is going to push over this branch I think he has everything figured out.

**Comment by corypride** [comment]
@chrissantillan what is supposed  to be rendered on the left nav and what dashboard is to be displayed when there are no feature level access boxes checked?

**Comment by corypride** [comment]
1. 0 features selected = admin-dashboard
                                        - Dashboard 
                                        - Students
                                        - Admins
                                        - Resources
                                        - Facilities

                                    2. 1 Open Content = open-content-dashboard
                                        - Dashboard
                                        - Open Content
                                        - Resources
                                        - Students
                                        - Admins
                                        - Facilities

                                    3. 1 Provider Platform Integrations = admin-dashboard
                                        - Student Activity = admin-dashboard
                                        - Learning Platforms
                                        - Students
                                        - Admins
                                        - Facilities

                                    4. 1 Program Management = ??? (admin dashboard for now)
                                        - Dashboard
                                        - Programs
                                        - Resources
                                        - Students
                                        - Admins
                                        - Facilities

                                    Now comes the first of the scenarios where there are two feature level access items selected
                                    In this case and as it was explained to me by Chris 12-3-24 at our 11am meeting, the dashboard rendered and the link that is at the
                                     top of the left-nav will be the dashboard of the feature level with the highest number 
                                     (1 - open-content, 2 - provider platforms, 3 - programs),we will see 
                                    the following: 

                                    5. 2 Open Content && Provider Platforms = admin-dashboard (one day these may all have their own version of a dashboard/landing-pg)
                                        - Dashboard
                                        - Learning Platforms
                                        - Open Content
                                        - Resources
                                        - Students
                                        - Admins                                 
                                        - Facilities

                                    6. 2 Open Content && Programs = admin-dashboard
                                        - Dashboard 
                                        - Programs
                                        - Open Content
                                        - Resources
                                        - Students
                                        - Admins                                        
                                        - Facilities
                                    
                                    7. 2 Provider Platforms && Programs = admin-dashboard
                                        - Dashboard
                                        - Programs
                                        - Learning Platforms
                                        - Resources                                        
                                        - Students
                                        - Admins
                                        - Facilities  

                                    8. 3 All feature levels selected
                                        - Dashboard
                                        - Programs
                                        - Learning Platforms      
                                        - Open Content
                                        - Resources
                                        - Students
                                        - Admins                                        
                                        - Facilities

**Comment by chrissantillan** [comment]
@corypride when no layers are selected, a Layer 0 dashboard (Operational Insights) is displayed. It's the issue that Preston is working on

File: frontend/src/Components/Navbar.tsx
Comment (PThorpe92) [inline frontend/src/Components/Navbar.tsx:100]: Is this rendering the link to the knowledge-center-management if there are no features enabled?

Comment (corypride) [reply frontend/src/Components/Navbar.tsx:100]: @PThorpe92  I don't understand what you are asking?

Comment (jtucholski) [reply frontend/src/Components/Navbar.tsx:100]: @corypride Resource Management was moved over to the Knowledge Center and is now called Helpful Links (#526)

**Comment by PThorpe92** [comment]
Ok @chrissantillan  we can hold off on this until #546 is merged then?

care to take a peek at that?

**Comment by PThorpe92** [comment]
also wraps up #541 by making username immutable/removing from the form to update

**Review by PThorpe92 — APPROVED** [review]

**Pull request merged**
